schemas/questionnaire_v1.json

```diff
@@ -59,14 +59,14 @@
         "StatisticsOfTradeAct"
       ]
     },
-    "view_submitted_responses": {
+    "view_submitted_response": {
       "type": "object",
       "properties": {
         "enabled": {
           "description": "Whether or not answers are available after submission.",
           "type": "boolean"
         },
-        "timeout": {
+        "duration": {
```

Comment:
@dwyeradam this could be named `view_submitted_reponse_timeout`?

Reply:
There isn't much point in concatenating the names if they are nested?